2023-12-03 23:09:24,647 INFO    StreamThr :4139 [internal.py:wandb_internal():86] W&B internal server running at pid: 4139, started at: 2023-12-03 23:09:24.647121
2023-12-03 23:09:24,649 DEBUG   HandlerThread:4139 [handler.py:handle_request():146] handle_request: status
2023-12-03 23:09:24,652 INFO    WriterThread:4139 [datastore.py:open_for_write():85] open: /root/dev/Merlion/wandb/run-20231203_230924-Best-AutoEncoder-ts_0-2023-12-03_23-09-22/run-Best-AutoEncoder-ts_0-2023-12-03_23-09-22.wandb
2023-12-03 23:09:24,653 DEBUG   SenderThread:4139 [sender.py:send():380] send: header
2023-12-03 23:09:24,665 DEBUG   SenderThread:4139 [sender.py:send():380] send: run
2023-12-03 23:09:24,667 INFO    SenderThread:4139 [sender.py:_maybe_setup_resume():759] checking resume status for None/MerlionAnomalyDetector/Best-AutoEncoder-ts_0-2023-12-03_23-09-22
2023-12-03 23:09:25,075 INFO    SenderThread:4139 [dir_watcher.py:__init__():211] watching files in: /root/dev/Merlion/wandb/run-20231203_230924-Best-AutoEncoder-ts_0-2023-12-03_23-09-22/files
2023-12-03 23:09:25,075 INFO    SenderThread:4139 [sender.py:_start_run_threads():1122] run started: Best-AutoEncoder-ts_0-2023-12-03_23-09-22 with start time 1701662964.647281
2023-12-03 23:09:25,083 DEBUG   HandlerThread:4139 [handler.py:handle_request():146] handle_request: check_version
2023-12-03 23:09:25,084 DEBUG   SenderThread:4139 [sender.py:send_request():407] send_request: check_version
2023-12-03 23:09:25,126 DEBUG   HandlerThread:4139 [handler.py:handle_request():146] handle_request: run_start
2023-12-03 23:09:25,167 DEBUG   HandlerThread:4139 [system_info.py:__init__():32] System info init
2023-12-03 23:09:25,168 DEBUG   HandlerThread:4139 [system_info.py:__init__():47] System info init done
2023-12-03 23:09:25,168 INFO    HandlerThread:4139 [system_monitor.py:start():194] Starting system monitor
2023-12-03 23:09:25,168 INFO    SystemMonitor:4139 [system_monitor.py:_start():158] Starting system asset monitoring threads
2023-12-03 23:09:25,168 INFO    HandlerThread:4139 [system_monitor.py:probe():214] Collecting system info
2023-12-03 23:09:25,169 INFO    SystemMonitor:4139 [interfaces.py:start():190] Started cpu monitoring
2023-12-03 23:09:25,170 INFO    SystemMonitor:4139 [interfaces.py:start():190] Started disk monitoring
2023-12-03 23:09:25,171 INFO    SystemMonitor:4139 [interfaces.py:start():190] Started gpu monitoring
2023-12-03 23:09:25,172 INFO    SystemMonitor:4139 [interfaces.py:start():190] Started memory monitoring
2023-12-03 23:09:25,173 INFO    SystemMonitor:4139 [interfaces.py:start():190] Started network monitoring
2023-12-03 23:09:25,187 DEBUG   HandlerThread:4139 [system_info.py:probe():196] Probing system
2023-12-03 23:09:25,230 DEBUG   HandlerThread:4139 [system_info.py:_probe_git():181] Probing git
2023-12-03 23:09:25,240 DEBUG   HandlerThread:4139 [system_info.py:_probe_git():189] Probing git done
2023-12-03 23:09:25,240 DEBUG   HandlerThread:4139 [system_info.py:probe():244] Probing system done
2023-12-03 23:09:25,240 DEBUG   HandlerThread:4139 [system_monitor.py:probe():223] {'os': 'Linux-5.15.133.1-microsoft-standard-WSL2-x86_64-with-glibc2.36', 'python': '3.12.0', 'heartbeatAt': '2023-12-04T04:09:25.187358', 'startedAt': '2023-12-04T04:09:24.641715', 'docker': None, 'cuda': None, 'args': (), 'state': 'running', 'program': '/root/dev/Merlion/maneuvers-adaptation/ManeuverDetection copy.py', 'codePathLocal': 'maneuvers-adaptation/ManeuverDetection copy.py', 'codePath': 'maneuvers-adaptation/ManeuverDetection copy.py', 'git': {'remote': 'git@github.com:Anosen/Merlion.git', 'commit': '180d9587d6b685bd704037a45b93b57d4bc59a9d'}, 'email': 'gregoire.marie08@gmail.com', 'root': '/root/dev/Merlion', 'host': 'MSI', 'username': 'root', 'executable': '/root/miniconda3/envs/sys843_env/bin/python', 'cpu_count': 6, 'cpu_count_logical': 12, 'cpu_freq': {'current': 2592.0100000000007, 'min': 0.0, 'max': 0.0}, 'cpu_freq_per_core': [{'current': 2592.01, 'min': 0.0, 'max': 0.0}, {'current': 2592.01, 'min': 0.0, 'max': 0.0}, {'current': 2592.01, 'min': 0.0, 'max': 0.0}, {'current': 2592.01, 'min': 0.0, 'max': 0.0}, {'current': 2592.01, 'min': 0.0, 'max': 0.0}, {'current': 2592.01, 'min': 0.0, 'max': 0.0}, {'current': 2592.01, 'min': 0.0, 'max': 0.0}, {'current': 2592.01, 'min': 0.0, 'max': 0.0}, {'current': 2592.01, 'min': 0.0, 'max': 0.0}, {'current': 2592.01, 'min': 0.0, 'max': 0.0}, {'current': 2592.01, 'min': 0.0, 'max': 0.0}, {'current': 2592.01, 'min': 0.0, 'max': 0.0}], 'disk': {'/': {'total': 1006.853931427002, 'used': 17.693790435791016}}, 'gpu': 'NVIDIA GeForce RTX 3060 Laptop GPU', 'gpu_count': 1, 'gpu_devices': [{'name': 'NVIDIA GeForce RTX 3060 Laptop GPU', 'memory_total': 6442450944}], 'memory': {'total': 7.671039581298828}}
2023-12-03 23:09:25,240 INFO    HandlerThread:4139 [system_monitor.py:probe():224] Finished collecting system info
2023-12-03 23:09:25,240 INFO    HandlerThread:4139 [system_monitor.py:probe():227] Publishing system info
2023-12-03 23:09:25,240 DEBUG   HandlerThread:4139 [system_info.py:_save_pip():52] Saving list of pip packages installed into the current environment
2023-12-03 23:09:25,242 DEBUG   HandlerThread:4139 [system_info.py:_save_pip():68] Saving pip packages done
2023-12-03 23:09:25,242 DEBUG   HandlerThread:4139 [system_info.py:_save_conda():75] Saving list of conda packages installed into the current environment
2023-12-03 23:09:26,076 INFO    Thread-12 :4139 [dir_watcher.py:_on_file_created():271] file/dir created: /root/dev/Merlion/wandb/run-20231203_230924-Best-AutoEncoder-ts_0-2023-12-03_23-09-22/files/conda-environment.yaml
2023-12-03 23:09:26,077 INFO    Thread-12 :4139 [dir_watcher.py:_on_file_created():271] file/dir created: /root/dev/Merlion/wandb/run-20231203_230924-Best-AutoEncoder-ts_0-2023-12-03_23-09-22/files/requirements.txt
2023-12-03 23:09:32,076 DEBUG   HandlerThread:4139 [system_info.py:_save_conda():87] Saving conda packages done
2023-12-03 23:09:32,077 INFO    HandlerThread:4139 [system_monitor.py:probe():229] Finished publishing system info
2023-12-03 23:09:32,077 INFO    Thread-12 :4139 [dir_watcher.py:_on_file_modified():288] file/dir modified: /root/dev/Merlion/wandb/run-20231203_230924-Best-AutoEncoder-ts_0-2023-12-03_23-09-22/files/conda-environment.yaml
2023-12-03 23:09:32,078 INFO    Thread-12 :4139 [dir_watcher.py:_on_file_created():271] file/dir created: /root/dev/Merlion/wandb/run-20231203_230924-Best-AutoEncoder-ts_0-2023-12-03_23-09-22/files/wandb-metadata.json
2023-12-03 23:09:32,080 DEBUG   HandlerThread:4139 [handler.py:handle_request():146] handle_request: status_report
2023-12-03 23:09:32,080 DEBUG   HandlerThread:4139 [handler.py:handle_request():146] handle_request: keepalive
2023-12-03 23:09:32,081 DEBUG   SenderThread:4139 [sender.py:send():380] send: files
2023-12-03 23:09:32,081 INFO    SenderThread:4139 [sender.py:_save_file():1378] saving file wandb-metadata.json with policy now
2023-12-03 23:09:32,084 DEBUG   HandlerThread:4139 [handler.py:handle_request():146] handle_request: stop_status
2023-12-03 23:09:32,085 DEBUG   SenderThread:4139 [sender.py:send_request():407] send_request: stop_status
2023-12-03 23:09:32,092 DEBUG   HandlerThread:4139 [handler.py:handle_request():146] handle_request: internal_messages
2023-12-03 23:09:32,236 DEBUG   SenderThread:4139 [sender.py:send():380] send: telemetry
2023-12-03 23:09:32,343 INFO    wandb-upload_0:4139 [upload_job.py:push():131] Uploaded file /tmp/tmp9h1lxm5rwandb/lqal0ctk-wandb-metadata.json
2023-12-03 23:09:33,077 INFO    Thread-12 :4139 [dir_watcher.py:_on_file_created():271] file/dir created: /root/dev/Merlion/wandb/run-20231203_230924-Best-AutoEncoder-ts_0-2023-12-03_23-09-22/files/output.log
2023-12-03 23:09:33,650 DEBUG   HandlerThread:4139 [handler.py:handle_request():146] handle_request: partial_history
2023-12-03 23:09:33,654 DEBUG   SenderThread:4139 [sender.py:send():380] send: history
2023-12-03 23:09:33,654 DEBUG   SenderThread:4139 [sender.py:send_request():407] send_request: summary_record
2023-12-03 23:09:33,656 INFO    SenderThread:4139 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2023-12-03 23:09:34,078 INFO    Thread-12 :4139 [dir_watcher.py:_on_file_created():271] file/dir created: /root/dev/Merlion/wandb/run-20231203_230924-Best-AutoEncoder-ts_0-2023-12-03_23-09-22/files/wandb-summary.json
2023-12-03 23:09:34,385 DEBUG   HandlerThread:4139 [handler.py:handle_request():146] handle_request: partial_history
2023-12-03 23:09:34,386 DEBUG   SenderThread:4139 [sender.py:send():380] send: history
2023-12-03 23:09:34,386 DEBUG   SenderThread:4139 [sender.py:send_request():407] send_request: summary_record
2023-12-03 23:09:34,387 INFO    SenderThread:4139 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2023-12-03 23:09:35,078 INFO    Thread-12 :4139 [dir_watcher.py:_on_file_modified():288] file/dir modified: /root/dev/Merlion/wandb/run-20231203_230924-Best-AutoEncoder-ts_0-2023-12-03_23-09-22/files/wandb-summary.json
2023-12-03 23:09:35,078 INFO    Thread-12 :4139 [dir_watcher.py:_on_file_modified():288] file/dir modified: /root/dev/Merlion/wandb/run-20231203_230924-Best-AutoEncoder-ts_0-2023-12-03_23-09-22/files/output.log
2023-12-03 23:09:35,100 DEBUG   HandlerThread:4139 [handler.py:handle_request():146] handle_request: partial_history
2023-12-03 23:09:35,101 DEBUG   SenderThread:4139 [sender.py:send():380] send: history
2023-12-03 23:09:35,101 DEBUG   SenderThread:4139 [sender.py:send_request():407] send_request: summary_record
2023-12-03 23:09:35,102 INFO    SenderThread:4139 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2023-12-03 23:09:35,839 DEBUG   HandlerThread:4139 [handler.py:handle_request():146] handle_request: status_report
2023-12-03 23:09:35,846 DEBUG   HandlerThread:4139 [handler.py:handle_request():146] handle_request: partial_history
2023-12-03 23:09:35,847 DEBUG   SenderThread:4139 [sender.py:send():380] send: history
2023-12-03 23:09:35,847 DEBUG   SenderThread:4139 [sender.py:send_request():407] send_request: summary_record
2023-12-03 23:09:35,849 INFO    SenderThread:4139 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2023-12-03 23:09:36,078 INFO    Thread-12 :4139 [dir_watcher.py:_on_file_modified():288] file/dir modified: /root/dev/Merlion/wandb/run-20231203_230924-Best-AutoEncoder-ts_0-2023-12-03_23-09-22/files/wandb-summary.json
2023-12-03 23:09:36,626 DEBUG   HandlerThread:4139 [handler.py:handle_request():146] handle_request: partial_history
2023-12-03 23:09:36,627 DEBUG   SenderThread:4139 [sender.py:send():380] send: history
2023-12-03 23:09:36,627 DEBUG   SenderThread:4139 [sender.py:send_request():407] send_request: summary_record
2023-12-03 23:09:36,628 INFO    SenderThread:4139 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2023-12-03 23:09:37,079 INFO    Thread-12 :4139 [dir_watcher.py:_on_file_modified():288] file/dir modified: /root/dev/Merlion/wandb/run-20231203_230924-Best-AutoEncoder-ts_0-2023-12-03_23-09-22/files/wandb-summary.json
2023-12-03 23:09:37,079 INFO    Thread-12 :4139 [dir_watcher.py:_on_file_modified():288] file/dir modified: /root/dev/Merlion/wandb/run-20231203_230924-Best-AutoEncoder-ts_0-2023-12-03_23-09-22/files/output.log
2023-12-03 23:09:37,345 DEBUG   HandlerThread:4139 [handler.py:handle_request():146] handle_request: partial_history
2023-12-03 23:09:37,347 DEBUG   SenderThread:4139 [sender.py:send():380] send: history
2023-12-03 23:09:37,347 DEBUG   SenderThread:4139 [sender.py:send_request():407] send_request: summary_record
2023-12-03 23:09:37,347 INFO    SenderThread:4139 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2023-12-03 23:09:38,079 INFO    Thread-12 :4139 [dir_watcher.py:_on_file_modified():288] file/dir modified: /root/dev/Merlion/wandb/run-20231203_230924-Best-AutoEncoder-ts_0-2023-12-03_23-09-22/files/wandb-summary.json
2023-12-03 23:09:38,189 DEBUG   HandlerThread:4139 [handler.py:handle_request():146] handle_request: partial_history
2023-12-03 23:09:38,191 DEBUG   SenderThread:4139 [sender.py:send():380] send: history
2023-12-03 23:09:38,191 DEBUG   SenderThread:4139 [sender.py:send_request():407] send_request: summary_record
2023-12-03 23:09:38,194 INFO    SenderThread:4139 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2023-12-03 23:09:39,075 DEBUG   HandlerThread:4139 [handler.py:handle_request():146] handle_request: partial_history
2023-12-03 23:09:39,076 DEBUG   SenderThread:4139 [sender.py:send():380] send: history
2023-12-03 23:09:39,077 DEBUG   SenderThread:4139 [sender.py:send_request():407] send_request: summary_record
2023-12-03 23:09:39,078 INFO    SenderThread:4139 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2023-12-03 23:09:39,079 INFO    Thread-12 :4139 [dir_watcher.py:_on_file_modified():288] file/dir modified: /root/dev/Merlion/wandb/run-20231203_230924-Best-AutoEncoder-ts_0-2023-12-03_23-09-22/files/wandb-summary.json
2023-12-03 23:09:39,080 INFO    Thread-12 :4139 [dir_watcher.py:_on_file_modified():288] file/dir modified: /root/dev/Merlion/wandb/run-20231203_230924-Best-AutoEncoder-ts_0-2023-12-03_23-09-22/files/output.log
2023-12-03 23:09:39,857 DEBUG   HandlerThread:4139 [handler.py:handle_request():146] handle_request: partial_history
2023-12-03 23:09:39,857 DEBUG   SenderThread:4139 [sender.py:send():380] send: history
2023-12-03 23:09:39,858 DEBUG   SenderThread:4139 [sender.py:send_request():407] send_request: summary_record
2023-12-03 23:09:39,858 INFO    SenderThread:4139 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2023-12-03 23:09:40,080 INFO    Thread-12 :4139 [dir_watcher.py:_on_file_modified():288] file/dir modified: /root/dev/Merlion/wandb/run-20231203_230924-Best-AutoEncoder-ts_0-2023-12-03_23-09-22/files/wandb-summary.json
2023-12-03 23:09:40,506 DEBUG   HandlerThread:4139 [handler.py:handle_request():146] handle_request: partial_history
2023-12-03 23:09:40,507 DEBUG   SenderThread:4139 [sender.py:send():380] send: history
2023-12-03 23:09:40,507 DEBUG   SenderThread:4139 [sender.py:send_request():407] send_request: summary_record
2023-12-03 23:09:40,507 INFO    SenderThread:4139 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2023-12-03 23:09:41,080 INFO    Thread-12 :4139 [dir_watcher.py:_on_file_modified():288] file/dir modified: /root/dev/Merlion/wandb/run-20231203_230924-Best-AutoEncoder-ts_0-2023-12-03_23-09-22/files/wandb-summary.json
2023-12-03 23:09:41,080 INFO    Thread-12 :4139 [dir_watcher.py:_on_file_modified():288] file/dir modified: /root/dev/Merlion/wandb/run-20231203_230924-Best-AutoEncoder-ts_0-2023-12-03_23-09-22/files/output.log
2023-12-03 23:09:41,172 DEBUG   HandlerThread:4139 [handler.py:handle_request():146] handle_request: status_report
2023-12-03 23:09:41,180 DEBUG   HandlerThread:4139 [handler.py:handle_request():146] handle_request: partial_history
2023-12-03 23:09:41,181 DEBUG   SenderThread:4139 [sender.py:send():380] send: history
2023-12-03 23:09:41,181 DEBUG   SenderThread:4139 [sender.py:send_request():407] send_request: summary_record
2023-12-03 23:09:41,182 INFO    SenderThread:4139 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2023-12-03 23:09:41,938 DEBUG   HandlerThread:4139 [handler.py:handle_request():146] handle_request: partial_history
2023-12-03 23:09:41,939 DEBUG   SenderThread:4139 [sender.py:send():380] send: history
2023-12-03 23:09:41,940 DEBUG   SenderThread:4139 [sender.py:send_request():407] send_request: summary_record
2023-12-03 23:09:41,940 INFO    SenderThread:4139 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2023-12-03 23:09:42,080 INFO    Thread-12 :4139 [dir_watcher.py:_on_file_modified():288] file/dir modified: /root/dev/Merlion/wandb/run-20231203_230924-Best-AutoEncoder-ts_0-2023-12-03_23-09-22/files/wandb-summary.json
2023-12-03 23:09:42,647 DEBUG   HandlerThread:4139 [handler.py:handle_request():146] handle_request: partial_history
2023-12-03 23:09:42,648 DEBUG   SenderThread:4139 [sender.py:send():380] send: history
2023-12-03 23:09:42,649 DEBUG   SenderThread:4139 [sender.py:send_request():407] send_request: summary_record
2023-12-03 23:09:42,649 INFO    SenderThread:4139 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2023-12-03 23:09:43,081 INFO    Thread-12 :4139 [dir_watcher.py:_on_file_modified():288] file/dir modified: /root/dev/Merlion/wandb/run-20231203_230924-Best-AutoEncoder-ts_0-2023-12-03_23-09-22/files/wandb-summary.json
2023-12-03 23:09:43,081 INFO    Thread-12 :4139 [dir_watcher.py:_on_file_modified():288] file/dir modified: /root/dev/Merlion/wandb/run-20231203_230924-Best-AutoEncoder-ts_0-2023-12-03_23-09-22/files/output.log
2023-12-03 23:09:43,313 DEBUG   HandlerThread:4139 [handler.py:handle_request():146] handle_request: partial_history
2023-12-03 23:09:43,314 DEBUG   SenderThread:4139 [sender.py:send():380] send: history
2023-12-03 23:09:43,314 DEBUG   SenderThread:4139 [sender.py:send_request():407] send_request: summary_record
2023-12-03 23:09:43,315 INFO    SenderThread:4139 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2023-12-03 23:09:44,081 INFO    Thread-12 :4139 [dir_watcher.py:_on_file_modified():288] file/dir modified: /root/dev/Merlion/wandb/run-20231203_230924-Best-AutoEncoder-ts_0-2023-12-03_23-09-22/files/wandb-summary.json
2023-12-03 23:09:44,111 DEBUG   HandlerThread:4139 [handler.py:handle_request():146] handle_request: partial_history
2023-12-03 23:09:44,112 DEBUG   SenderThread:4139 [sender.py:send():380] send: history
2023-12-03 23:09:44,112 DEBUG   SenderThread:4139 [sender.py:send_request():407] send_request: summary_record
2023-12-03 23:09:44,112 INFO    SenderThread:4139 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2023-12-03 23:09:44,974 DEBUG   HandlerThread:4139 [handler.py:handle_request():146] handle_request: partial_history
2023-12-03 23:09:44,975 DEBUG   SenderThread:4139 [sender.py:send():380] send: history
2023-12-03 23:09:44,976 DEBUG   SenderThread:4139 [sender.py:send_request():407] send_request: summary_record
2023-12-03 23:09:44,976 INFO    SenderThread:4139 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2023-12-03 23:09:45,081 INFO    Thread-12 :4139 [dir_watcher.py:_on_file_modified():288] file/dir modified: /root/dev/Merlion/wandb/run-20231203_230924-Best-AutoEncoder-ts_0-2023-12-03_23-09-22/files/wandb-summary.json
2023-12-03 23:09:45,082 INFO    Thread-12 :4139 [dir_watcher.py:_on_file_modified():288] file/dir modified: /root/dev/Merlion/wandb/run-20231203_230924-Best-AutoEncoder-ts_0-2023-12-03_23-09-22/files/output.log
2023-12-03 23:09:45,726 DEBUG   HandlerThread:4139 [handler.py:handle_request():146] handle_request: partial_history
2023-12-03 23:09:45,729 DEBUG   SenderThread:4139 [sender.py:send():380] send: history
2023-12-03 23:09:45,729 DEBUG   SenderThread:4139 [sender.py:send_request():407] send_request: summary_record
2023-12-03 23:09:45,729 INFO    SenderThread:4139 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2023-12-03 23:09:46,082 INFO    Thread-12 :4139 [dir_watcher.py:_on_file_modified():288] file/dir modified: /root/dev/Merlion/wandb/run-20231203_230924-Best-AutoEncoder-ts_0-2023-12-03_23-09-22/files/wandb-summary.json
2023-12-03 23:09:46,503 DEBUG   HandlerThread:4139 [handler.py:handle_request():146] handle_request: status_report
2023-12-03 23:09:46,512 DEBUG   HandlerThread:4139 [handler.py:handle_request():146] handle_request: partial_history
2023-12-03 23:09:46,513 DEBUG   SenderThread:4139 [sender.py:send():380] send: history
2023-12-03 23:09:46,513 DEBUG   SenderThread:4139 [sender.py:send_request():407] send_request: summary_record
2023-12-03 23:09:46,513 INFO    SenderThread:4139 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2023-12-03 23:09:47,082 INFO    Thread-12 :4139 [dir_watcher.py:_on_file_modified():288] file/dir modified: /root/dev/Merlion/wandb/run-20231203_230924-Best-AutoEncoder-ts_0-2023-12-03_23-09-22/files/wandb-summary.json
2023-12-03 23:09:47,082 INFO    Thread-12 :4139 [dir_watcher.py:_on_file_modified():288] file/dir modified: /root/dev/Merlion/wandb/run-20231203_230924-Best-AutoEncoder-ts_0-2023-12-03_23-09-22/files/output.log
2023-12-03 23:09:47,103 DEBUG   HandlerThread:4139 [handler.py:handle_request():146] handle_request: internal_messages
2023-12-03 23:09:47,103 DEBUG   HandlerThread:4139 [handler.py:handle_request():146] handle_request: stop_status
2023-12-03 23:09:47,104 DEBUG   SenderThread:4139 [sender.py:send_request():407] send_request: stop_status
2023-12-03 23:09:47,387 DEBUG   HandlerThread:4139 [handler.py:handle_request():146] handle_request: partial_history
2023-12-03 23:09:47,388 DEBUG   SenderThread:4139 [sender.py:send():380] send: history
2023-12-03 23:09:47,388 DEBUG   SenderThread:4139 [sender.py:send_request():407] send_request: summary_record
2023-12-03 23:09:47,388 INFO    SenderThread:4139 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2023-12-03 23:09:47,819 DEBUG   SenderThread:4139 [sender.py:send():380] send: exit
2023-12-03 23:09:47,819 INFO    SenderThread:4139 [sender.py:send_exit():585] handling exit code: 255
2023-12-03 23:09:47,819 INFO    SenderThread:4139 [sender.py:send_exit():587] handling runtime: 22
2023-12-03 23:09:47,820 INFO    SenderThread:4139 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2023-12-03 23:09:47,820 INFO    SenderThread:4139 [sender.py:send_exit():593] send defer
2023-12-03 23:09:47,820 DEBUG   HandlerThread:4139 [handler.py:handle_request():146] handle_request: defer
2023-12-03 23:09:47,820 INFO    HandlerThread:4139 [handler.py:handle_request_defer():172] handle defer: 0
2023-12-03 23:09:47,821 DEBUG   SenderThread:4139 [sender.py:send_request():407] send_request: defer
2023-12-03 23:09:47,821 INFO    SenderThread:4139 [sender.py:send_request_defer():609] handle sender defer: 0
2023-12-03 23:09:47,821 INFO    SenderThread:4139 [sender.py:transition_state():613] send defer: 1
2023-12-03 23:09:47,821 DEBUG   HandlerThread:4139 [handler.py:handle_request():146] handle_request: defer
2023-12-03 23:09:47,821 INFO    HandlerThread:4139 [handler.py:handle_request_defer():172] handle defer: 1
2023-12-03 23:09:47,822 DEBUG   SenderThread:4139 [sender.py:send_request():407] send_request: defer
2023-12-03 23:09:47,822 INFO    SenderThread:4139 [sender.py:send_request_defer():609] handle sender defer: 1
2023-12-03 23:09:47,822 INFO    SenderThread:4139 [sender.py:transition_state():613] send defer: 2
2023-12-03 23:09:47,822 DEBUG   HandlerThread:4139 [handler.py:handle_request():146] handle_request: defer
2023-12-03 23:09:47,822 INFO    HandlerThread:4139 [handler.py:handle_request_defer():172] handle defer: 2
2023-12-03 23:09:47,822 INFO    HandlerThread:4139 [system_monitor.py:finish():203] Stopping system monitor
2023-12-03 23:09:47,822 DEBUG   SystemMonitor:4139 [system_monitor.py:_start():172] Starting system metrics aggregation loop
2023-12-03 23:09:47,823 DEBUG   SystemMonitor:4139 [system_monitor.py:_start():179] Finished system metrics aggregation loop
2023-12-03 23:09:47,823 INFO    HandlerThread:4139 [interfaces.py:finish():202] Joined cpu monitor
2023-12-03 23:09:47,824 DEBUG   SystemMonitor:4139 [system_monitor.py:_start():183] Publishing last batch of metrics
2023-12-03 23:09:47,824 INFO    HandlerThread:4139 [interfaces.py:finish():202] Joined disk monitor
2023-12-03 23:09:47,842 INFO    HandlerThread:4139 [interfaces.py:finish():202] Joined gpu monitor
2023-12-03 23:09:47,842 INFO    HandlerThread:4139 [interfaces.py:finish():202] Joined memory monitor
2023-12-03 23:09:47,842 INFO    HandlerThread:4139 [interfaces.py:finish():202] Joined network monitor
2023-12-03 23:09:47,843 DEBUG   SenderThread:4139 [sender.py:send_request():407] send_request: defer
2023-12-03 23:09:47,843 INFO    SenderThread:4139 [sender.py:send_request_defer():609] handle sender defer: 2
2023-12-03 23:09:47,843 INFO    SenderThread:4139 [sender.py:transition_state():613] send defer: 3
2023-12-03 23:09:47,843 DEBUG   SenderThread:4139 [sender.py:send():380] send: stats
2023-12-03 23:09:47,843 DEBUG   HandlerThread:4139 [handler.py:handle_request():146] handle_request: defer
2023-12-03 23:09:47,844 INFO    HandlerThread:4139 [handler.py:handle_request_defer():172] handle defer: 3
2023-12-03 23:09:47,844 DEBUG   SenderThread:4139 [sender.py:send_request():407] send_request: defer
2023-12-03 23:09:47,845 INFO    SenderThread:4139 [sender.py:send_request_defer():609] handle sender defer: 3
2023-12-03 23:09:47,845 INFO    SenderThread:4139 [sender.py:transition_state():613] send defer: 4
2023-12-03 23:09:47,845 DEBUG   HandlerThread:4139 [handler.py:handle_request():146] handle_request: defer
2023-12-03 23:09:47,845 INFO    HandlerThread:4139 [handler.py:handle_request_defer():172] handle defer: 4
2023-12-03 23:09:47,845 DEBUG   SenderThread:4139 [sender.py:send_request():407] send_request: defer
2023-12-03 23:09:47,845 INFO    SenderThread:4139 [sender.py:send_request_defer():609] handle sender defer: 4
2023-12-03 23:09:47,845 INFO    SenderThread:4139 [sender.py:transition_state():613] send defer: 5
2023-12-03 23:09:47,846 DEBUG   HandlerThread:4139 [handler.py:handle_request():146] handle_request: defer
2023-12-03 23:09:47,846 INFO    HandlerThread:4139 [handler.py:handle_request_defer():172] handle defer: 5
2023-12-03 23:09:47,846 DEBUG   SenderThread:4139 [sender.py:send():380] send: summary
2023-12-03 23:09:47,847 INFO    SenderThread:4139 [sender.py:_save_file():1378] saving file wandb-summary.json with policy end
2023-12-03 23:09:47,847 DEBUG   SenderThread:4139 [sender.py:send_request():407] send_request: defer
2023-12-03 23:09:47,847 INFO    SenderThread:4139 [sender.py:send_request_defer():609] handle sender defer: 5
2023-12-03 23:09:47,847 INFO    SenderThread:4139 [sender.py:transition_state():613] send defer: 6
2023-12-03 23:09:47,847 DEBUG   HandlerThread:4139 [handler.py:handle_request():146] handle_request: defer
2023-12-03 23:09:47,847 INFO    HandlerThread:4139 [handler.py:handle_request_defer():172] handle defer: 6
2023-12-03 23:09:47,848 DEBUG   SenderThread:4139 [sender.py:send_request():407] send_request: defer
2023-12-03 23:09:47,848 INFO    SenderThread:4139 [sender.py:send_request_defer():609] handle sender defer: 6
2023-12-03 23:09:47,852 DEBUG   HandlerThread:4139 [handler.py:handle_request():146] handle_request: status_report
2023-12-03 23:09:47,925 INFO    SenderThread:4139 [sender.py:transition_state():613] send defer: 7
2023-12-03 23:09:47,925 DEBUG   HandlerThread:4139 [handler.py:handle_request():146] handle_request: defer
2023-12-03 23:09:47,925 INFO    HandlerThread:4139 [handler.py:handle_request_defer():172] handle defer: 7
2023-12-03 23:09:47,925 DEBUG   SenderThread:4139 [sender.py:send_request():407] send_request: defer
2023-12-03 23:09:47,926 INFO    SenderThread:4139 [sender.py:send_request_defer():609] handle sender defer: 7
2023-12-03 23:09:48,082 INFO    Thread-12 :4139 [dir_watcher.py:_on_file_modified():288] file/dir modified: /root/dev/Merlion/wandb/run-20231203_230924-Best-AutoEncoder-ts_0-2023-12-03_23-09-22/files/wandb-summary.json
2023-12-03 23:09:48,082 INFO    Thread-12 :4139 [dir_watcher.py:_on_file_modified():288] file/dir modified: /root/dev/Merlion/wandb/run-20231203_230924-Best-AutoEncoder-ts_0-2023-12-03_23-09-22/files/config.yaml
2023-12-03 23:09:48,819 DEBUG   HandlerThread:4139 [handler.py:handle_request():146] handle_request: poll_exit
2023-12-03 23:09:49,082 INFO    Thread-12 :4139 [dir_watcher.py:_on_file_modified():288] file/dir modified: /root/dev/Merlion/wandb/run-20231203_230924-Best-AutoEncoder-ts_0-2023-12-03_23-09-22/files/output.log
2023-12-03 23:09:49,818 INFO    SenderThread:4139 [sender.py:transition_state():613] send defer: 8
2023-12-03 23:09:49,818 DEBUG   SenderThread:4139 [sender.py:send_request():407] send_request: poll_exit
2023-12-03 23:09:49,818 DEBUG   HandlerThread:4139 [handler.py:handle_request():146] handle_request: defer
2023-12-03 23:09:49,818 INFO    HandlerThread:4139 [handler.py:handle_request_defer():172] handle defer: 8
2023-12-03 23:09:49,819 DEBUG   SenderThread:4139 [sender.py:send_request():407] send_request: defer
2023-12-03 23:09:49,819 INFO    SenderThread:4139 [sender.py:send_request_defer():609] handle sender defer: 8
2023-12-03 23:09:49,819 INFO    SenderThread:4139 [job_builder.py:build():281] Attempting to build job artifact
2023-12-03 23:09:49,819 INFO    SenderThread:4139 [job_builder.py:_get_source_type():393] is repo sourced job
2023-12-03 23:09:49,821 DEBUG   HandlerThread:4139 [handler.py:handle_request():146] handle_request: poll_exit
2023-12-03 23:09:49,822 INFO    SenderThread:4139 [job_builder.py:build():369] adding wandb-job metadata file
2023-12-03 23:09:49,827 INFO    SenderThread:4139 [sender.py:transition_state():613] send defer: 9
2023-12-03 23:09:49,828 DEBUG   SenderThread:4139 [sender.py:send_request():407] send_request: poll_exit
2023-12-03 23:09:49,828 DEBUG   HandlerThread:4139 [handler.py:handle_request():146] handle_request: defer
2023-12-03 23:09:49,828 DEBUG   SenderThread:4139 [sender.py:send():380] send: artifact
2023-12-03 23:09:49,828 INFO    HandlerThread:4139 [handler.py:handle_request_defer():172] handle defer: 9
2023-12-03 23:09:50,083 INFO    Thread-12 :4139 [dir_watcher.py:_on_file_modified():288] file/dir modified: /root/dev/Merlion/wandb/run-20231203_230924-Best-AutoEncoder-ts_0-2023-12-03_23-09-22/files/output.log
2023-12-03 23:09:50,460 INFO    wandb-upload_0:4139 [upload_job.py:push():89] Uploaded file /root/.local/share/wandb/artifacts/staging/tmpv55jhej_
2023-12-03 23:09:50,462 INFO    wandb-upload_1:4139 [upload_job.py:push():89] Uploaded file /root/.local/share/wandb/artifacts/staging/tmpe4vy60zg
2023-12-03 23:09:50,821 DEBUG   HandlerThread:4139 [handler.py:handle_request():146] handle_request: poll_exit
2023-12-03 23:09:51,636 INFO    SenderThread:4139 [sender.py:send_artifact():1456] sent artifact job-git_github.com_Anosen_Merlion.git_maneuvers-adaptation_ManeuverDetection_copy.py - {'id': 'QXJ0aWZhY3Q6NjU2NTMyODI2', 'state': 'PENDING', 'artifactSequence': {'id': 'QXJ0aWZhY3RDb2xsZWN0aW9uOjEyMDYxNjI1Nw==', 'latestArtifact': None}}
2023-12-03 23:09:51,636 DEBUG   SenderThread:4139 [sender.py:send_request():407] send_request: defer
2023-12-03 23:09:51,636 INFO    SenderThread:4139 [sender.py:send_request_defer():609] handle sender defer: 9
2023-12-03 23:09:51,636 INFO    SenderThread:4139 [dir_watcher.py:finish():358] shutting down directory watcher
2023-12-03 23:09:52,084 INFO    SenderThread:4139 [dir_watcher.py:finish():388] scan: /root/dev/Merlion/wandb/run-20231203_230924-Best-AutoEncoder-ts_0-2023-12-03_23-09-22/files
2023-12-03 23:09:52,084 INFO    SenderThread:4139 [dir_watcher.py:finish():402] scan save: /root/dev/Merlion/wandb/run-20231203_230924-Best-AutoEncoder-ts_0-2023-12-03_23-09-22/files/wandb-metadata.json wandb-metadata.json
2023-12-03 23:09:52,084 INFO    SenderThread:4139 [dir_watcher.py:finish():402] scan save: /root/dev/Merlion/wandb/run-20231203_230924-Best-AutoEncoder-ts_0-2023-12-03_23-09-22/files/config.yaml config.yaml
2023-12-03 23:09:52,084 INFO    SenderThread:4139 [dir_watcher.py:finish():402] scan save: /root/dev/Merlion/wandb/run-20231203_230924-Best-AutoEncoder-ts_0-2023-12-03_23-09-22/files/conda-environment.yaml conda-environment.yaml
2023-12-03 23:09:52,084 INFO    SenderThread:4139 [dir_watcher.py:finish():402] scan save: /root/dev/Merlion/wandb/run-20231203_230924-Best-AutoEncoder-ts_0-2023-12-03_23-09-22/files/requirements.txt requirements.txt
2023-12-03 23:09:52,087 INFO    SenderThread:4139 [dir_watcher.py:finish():402] scan save: /root/dev/Merlion/wandb/run-20231203_230924-Best-AutoEncoder-ts_0-2023-12-03_23-09-22/files/output.log output.log
2023-12-03 23:09:52,088 INFO    SenderThread:4139 [dir_watcher.py:finish():402] scan save: /root/dev/Merlion/wandb/run-20231203_230924-Best-AutoEncoder-ts_0-2023-12-03_23-09-22/files/wandb-summary.json wandb-summary.json
2023-12-03 23:09:52,092 INFO    SenderThread:4139 [sender.py:transition_state():613] send defer: 10
2023-12-03 23:09:52,092 DEBUG   SenderThread:4139 [sender.py:send_request():407] send_request: poll_exit
2023-12-03 23:09:52,094 DEBUG   HandlerThread:4139 [handler.py:handle_request():146] handle_request: defer
2023-12-03 23:09:52,095 INFO    HandlerThread:4139 [handler.py:handle_request_defer():172] handle defer: 10
2023-12-03 23:09:52,095 DEBUG   SenderThread:4139 [sender.py:send_request():407] send_request: defer
2023-12-03 23:09:52,095 INFO    SenderThread:4139 [sender.py:send_request_defer():609] handle sender defer: 10
2023-12-03 23:09:52,095 INFO    SenderThread:4139 [file_pusher.py:finish():175] shutting down file pusher
2023-12-03 23:09:52,149 ERROR   wandb-upload_0:4139 [internal_api.py:execute():369] 404 response executing GraphQL.
2023-12-03 23:09:52,150 ERROR   wandb-upload_1:4139 [internal_api.py:execute():369] 404 response executing GraphQL.
2023-12-03 23:09:52,150 ERROR   wandb-upload_0:4139 [internal_api.py:execute():370] {"errors":[{"message":"run MerlionAnomalyDetector/Best-AutoEncoder-ts_0-2023-12-03_23-09-22 not found during createRunFiles","path":["createRunFiles"]}],"data":{"createRunFiles":null}}
2023-12-03 23:09:52,150 ERROR   wandb-upload_1:4139 [internal_api.py:execute():370] {"errors":[{"message":"run MerlionAnomalyDetector/Best-AutoEncoder-ts_0-2023-12-03_23-09-22 not found during createRunFiles","path":["createRunFiles"]}],"data":{"createRunFiles":null}}
2023-12-03 23:09:52,151 ERROR   Thread-11 (_thread_body):4139 [step_upload.py:_handle_event():200] Failed to upload file: /root/dev/Merlion/wandb/run-20231203_230924-Best-AutoEncoder-ts_0-2023-12-03_23-09-22/files/config.yaml
Traceback (most recent call last):
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/sdk/internal/internal_api.py", line 366, in execute
    return self.client.execute(*args, **kwargs)  # type: ignore
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/vendor/gql-0.2.0/wandb_gql/client.py", line 52, in execute
    result = self._get_result(document, *args, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/vendor/gql-0.2.0/wandb_gql/client.py", line 60, in _get_result
    return self.transport.execute(document, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/sdk/lib/gql_request.py", line 59, in execute
    request.raise_for_status()
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://api.wandb.ai/graphql

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/concurrent/futures/thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/filesync/step_upload.py", line 288, in run_and_notify
    self._do_upload_sync(event)
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/filesync/step_upload.py", line 330, in _do_upload_sync
    job.run()
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/filesync/upload_job.py", line 57, in run
    self.push()
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/filesync/upload_job.py", line 106, in push
    _, upload_headers, result = self._api.upload_urls(project, [self.save_name])
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/apis/normalize.py", line 73, in wrapper
    raise err
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/apis/normalize.py", line 41, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/sdk/internal/internal_api.py", line 2175, in upload_urls
    query_result = self.gql(
                   ^^^^^^^^^
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/sdk/internal/internal_api.py", line 338, in gql
    ret = self._retry_gql(
          ^^^^^^^^^^^^^^^^
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/sdk/lib/retry.py", line 147, in __call__
    retry_timedelta_triggered = check_retry_fn(e)
                                ^^^^^^^^^^^^^^^^^
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/util.py", line 906, in no_retry_auth
    raise CommError(
wandb.errors.CommError: It appears that you do not have permission to access the requested resource. Please reach out to the project owner to grant you access. If you have the correct permissions, verify that there are no issues with your networking setup.(Error 404: Not Found)
2023-12-03 23:09:52,167 ERROR   Thread-11 (_thread_body):4139 [step_upload.py:_handle_event():200] Failed to upload file: /root/dev/Merlion/wandb/run-20231203_230924-Best-AutoEncoder-ts_0-2023-12-03_23-09-22/files/conda-environment.yaml
Traceback (most recent call last):
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/sdk/internal/internal_api.py", line 366, in execute
    return self.client.execute(*args, **kwargs)  # type: ignore
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/vendor/gql-0.2.0/wandb_gql/client.py", line 52, in execute
    result = self._get_result(document, *args, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/vendor/gql-0.2.0/wandb_gql/client.py", line 60, in _get_result
    return self.transport.execute(document, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/sdk/lib/gql_request.py", line 59, in execute
    request.raise_for_status()
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://api.wandb.ai/graphql

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/concurrent/futures/thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/filesync/step_upload.py", line 288, in run_and_notify
    self._do_upload_sync(event)
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/filesync/step_upload.py", line 330, in _do_upload_sync
    job.run()
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/filesync/upload_job.py", line 57, in run
    self.push()
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/filesync/upload_job.py", line 106, in push
    _, upload_headers, result = self._api.upload_urls(project, [self.save_name])
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/apis/normalize.py", line 73, in wrapper
    raise err
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/apis/normalize.py", line 41, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/sdk/internal/internal_api.py", line 2175, in upload_urls
    query_result = self.gql(
                   ^^^^^^^^^
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/sdk/internal/internal_api.py", line 338, in gql
    ret = self._retry_gql(
          ^^^^^^^^^^^^^^^^
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/sdk/lib/retry.py", line 147, in __call__
    retry_timedelta_triggered = check_retry_fn(e)
                                ^^^^^^^^^^^^^^^^^
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/util.py", line 906, in no_retry_auth
    raise CommError(
wandb.errors.CommError: It appears that you do not have permission to access the requested resource. Please reach out to the project owner to grant you access. If you have the correct permissions, verify that there are no issues with your networking setup.(Error 404: Not Found)
2023-12-03 23:09:52,222 ERROR   wandb-upload_2:4139 [internal_api.py:execute():369] 404 response executing GraphQL.
2023-12-03 23:09:52,222 ERROR   wandb-upload_2:4139 [internal_api.py:execute():370] {"errors":[{"message":"run MerlionAnomalyDetector/Best-AutoEncoder-ts_0-2023-12-03_23-09-22 not found during createRunFiles","path":["createRunFiles"]}],"data":{"createRunFiles":null}}
2023-12-03 23:09:52,223 ERROR   wandb-upload_4:4139 [internal_api.py:execute():369] 404 response executing GraphQL.
2023-12-03 23:09:52,223 ERROR   wandb-upload_4:4139 [internal_api.py:execute():370] {"errors":[{"message":"run MerlionAnomalyDetector/Best-AutoEncoder-ts_0-2023-12-03_23-09-22 not found during createRunFiles","path":["createRunFiles"]}],"data":{"createRunFiles":null}}
2023-12-03 23:09:52,223 ERROR   wandb-upload_3:4139 [internal_api.py:execute():369] 404 response executing GraphQL.
2023-12-03 23:09:52,223 ERROR   Thread-11 (_thread_body):4139 [step_upload.py:_handle_event():200] Failed to upload file: /root/dev/Merlion/wandb/run-20231203_230924-Best-AutoEncoder-ts_0-2023-12-03_23-09-22/files/requirements.txt
Traceback (most recent call last):
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/sdk/internal/internal_api.py", line 366, in execute
    return self.client.execute(*args, **kwargs)  # type: ignore
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/vendor/gql-0.2.0/wandb_gql/client.py", line 52, in execute
    result = self._get_result(document, *args, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/vendor/gql-0.2.0/wandb_gql/client.py", line 60, in _get_result
    return self.transport.execute(document, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/sdk/lib/gql_request.py", line 59, in execute
    request.raise_for_status()
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://api.wandb.ai/graphql

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/concurrent/futures/thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/filesync/step_upload.py", line 288, in run_and_notify
    self._do_upload_sync(event)
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/filesync/step_upload.py", line 330, in _do_upload_sync
    job.run()
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/filesync/upload_job.py", line 57, in run
    self.push()
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/filesync/upload_job.py", line 106, in push
    _, upload_headers, result = self._api.upload_urls(project, [self.save_name])
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/apis/normalize.py", line 73, in wrapper
    raise err
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/apis/normalize.py", line 41, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/sdk/internal/internal_api.py", line 2175, in upload_urls
    query_result = self.gql(
                   ^^^^^^^^^
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/sdk/internal/internal_api.py", line 338, in gql
    ret = self._retry_gql(
          ^^^^^^^^^^^^^^^^
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/sdk/lib/retry.py", line 147, in __call__
    retry_timedelta_triggered = check_retry_fn(e)
                                ^^^^^^^^^^^^^^^^^
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/util.py", line 906, in no_retry_auth
    raise CommError(
wandb.errors.CommError: It appears that you do not have permission to access the requested resource. Please reach out to the project owner to grant you access. If you have the correct permissions, verify that there are no issues with your networking setup.(Error 404: Not Found)
2023-12-03 23:09:52,224 ERROR   wandb-upload_3:4139 [internal_api.py:execute():370] {"errors":[{"message":"run MerlionAnomalyDetector/Best-AutoEncoder-ts_0-2023-12-03_23-09-22 not found during createRunFiles","path":["createRunFiles"]}],"data":{"createRunFiles":null}}
2023-12-03 23:09:52,225 ERROR   Thread-11 (_thread_body):4139 [step_upload.py:_handle_event():200] Failed to upload file: /root/dev/Merlion/wandb/run-20231203_230924-Best-AutoEncoder-ts_0-2023-12-03_23-09-22/files/wandb-summary.json
Traceback (most recent call last):
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/sdk/internal/internal_api.py", line 366, in execute
    return self.client.execute(*args, **kwargs)  # type: ignore
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/vendor/gql-0.2.0/wandb_gql/client.py", line 52, in execute
    result = self._get_result(document, *args, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/vendor/gql-0.2.0/wandb_gql/client.py", line 60, in _get_result
    return self.transport.execute(document, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/sdk/lib/gql_request.py", line 59, in execute
    request.raise_for_status()
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://api.wandb.ai/graphql

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/concurrent/futures/thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/filesync/step_upload.py", line 288, in run_and_notify
    self._do_upload_sync(event)
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/filesync/step_upload.py", line 330, in _do_upload_sync
    job.run()
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/filesync/upload_job.py", line 57, in run
    self.push()
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/filesync/upload_job.py", line 106, in push
    _, upload_headers, result = self._api.upload_urls(project, [self.save_name])
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/apis/normalize.py", line 73, in wrapper
    raise err
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/apis/normalize.py", line 41, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/sdk/internal/internal_api.py", line 2175, in upload_urls
    query_result = self.gql(
                   ^^^^^^^^^
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/sdk/internal/internal_api.py", line 338, in gql
    ret = self._retry_gql(
          ^^^^^^^^^^^^^^^^
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/sdk/lib/retry.py", line 147, in __call__
    retry_timedelta_triggered = check_retry_fn(e)
                                ^^^^^^^^^^^^^^^^^
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/util.py", line 906, in no_retry_auth
    raise CommError(
wandb.errors.CommError: It appears that you do not have permission to access the requested resource. Please reach out to the project owner to grant you access. If you have the correct permissions, verify that there are no issues with your networking setup.(Error 404: Not Found)
2023-12-03 23:09:52,226 ERROR   Thread-11 (_thread_body):4139 [step_upload.py:_handle_event():200] Failed to upload file: /root/dev/Merlion/wandb/run-20231203_230924-Best-AutoEncoder-ts_0-2023-12-03_23-09-22/files/output.log
Traceback (most recent call last):
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/sdk/internal/internal_api.py", line 366, in execute
    return self.client.execute(*args, **kwargs)  # type: ignore
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/vendor/gql-0.2.0/wandb_gql/client.py", line 52, in execute
    result = self._get_result(document, *args, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/vendor/gql-0.2.0/wandb_gql/client.py", line 60, in _get_result
    return self.transport.execute(document, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/sdk/lib/gql_request.py", line 59, in execute
    request.raise_for_status()
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://api.wandb.ai/graphql

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/concurrent/futures/thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/filesync/step_upload.py", line 288, in run_and_notify
    self._do_upload_sync(event)
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/filesync/step_upload.py", line 330, in _do_upload_sync
    job.run()
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/filesync/upload_job.py", line 57, in run
    self.push()
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/filesync/upload_job.py", line 106, in push
    _, upload_headers, result = self._api.upload_urls(project, [self.save_name])
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/apis/normalize.py", line 73, in wrapper
    raise err
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/apis/normalize.py", line 41, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/sdk/internal/internal_api.py", line 2175, in upload_urls
    query_result = self.gql(
                   ^^^^^^^^^
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/sdk/internal/internal_api.py", line 338, in gql
    ret = self._retry_gql(
          ^^^^^^^^^^^^^^^^
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/sdk/lib/retry.py", line 147, in __call__
    retry_timedelta_triggered = check_retry_fn(e)
                                ^^^^^^^^^^^^^^^^^
  File "/root/miniconda3/envs/sys843_env/lib/python3.12/site-packages/wandb/util.py", line 906, in no_retry_auth
    raise CommError(
wandb.errors.CommError: It appears that you do not have permission to access the requested resource. Please reach out to the project owner to grant you access. If you have the correct permissions, verify that there are no issues with your networking setup.(Error 404: Not Found)
2023-12-03 23:09:52,427 INFO    Thread-11 (_thread_body):4139 [sender.py:transition_state():613] send defer: 11
2023-12-03 23:09:52,428 DEBUG   HandlerThread:4139 [handler.py:handle_request():146] handle_request: defer
2023-12-03 23:09:52,428 INFO    HandlerThread:4139 [handler.py:handle_request_defer():172] handle defer: 11
2023-12-03 23:09:52,429 DEBUG   SenderThread:4139 [sender.py:send_request():407] send_request: defer
2023-12-03 23:09:52,429 INFO    SenderThread:4139 [sender.py:send_request_defer():609] handle sender defer: 11
2023-12-03 23:09:52,429 INFO    SenderThread:4139 [file_pusher.py:join():181] waiting for file pusher
2023-12-03 23:09:52,429 INFO    SenderThread:4139 [sender.py:transition_state():613] send defer: 12
2023-12-03 23:09:52,429 DEBUG   HandlerThread:4139 [handler.py:handle_request():146] handle_request: defer
2023-12-03 23:09:52,429 INFO    HandlerThread:4139 [handler.py:handle_request_defer():172] handle defer: 12
2023-12-03 23:09:52,429 DEBUG   SenderThread:4139 [sender.py:send_request():407] send_request: defer
2023-12-03 23:09:52,429 INFO    SenderThread:4139 [sender.py:send_request_defer():609] handle sender defer: 12
2023-12-03 23:09:52,430 INFO    SenderThread:4139 [file_stream.py:finish():595] file stream finish called
2023-12-03 23:09:52,470 INFO    SenderThread:4139 [file_stream.py:finish():599] file stream finish is done
2023-12-03 23:09:52,470 INFO    SenderThread:4139 [sender.py:transition_state():613] send defer: 13
2023-12-03 23:09:52,470 DEBUG   HandlerThread:4139 [handler.py:handle_request():146] handle_request: defer
2023-12-03 23:09:52,470 INFO    HandlerThread:4139 [handler.py:handle_request_defer():172] handle defer: 13
2023-12-03 23:09:52,471 DEBUG   SenderThread:4139 [sender.py:send_request():407] send_request: defer
2023-12-03 23:09:52,471 INFO    SenderThread:4139 [sender.py:send_request_defer():609] handle sender defer: 13
2023-12-03 23:09:52,471 INFO    SenderThread:4139 [sender.py:transition_state():613] send defer: 14
2023-12-03 23:09:52,471 DEBUG   HandlerThread:4139 [handler.py:handle_request():146] handle_request: defer
2023-12-03 23:09:52,472 INFO    HandlerThread:4139 [handler.py:handle_request_defer():172] handle defer: 14
2023-12-03 23:09:52,473 DEBUG   SenderThread:4139 [sender.py:send():380] send: final
2023-12-03 23:09:52,473 DEBUG   SenderThread:4139 [sender.py:send():380] send: footer
2023-12-03 23:09:52,473 DEBUG   SenderThread:4139 [sender.py:send_request():407] send_request: defer
2023-12-03 23:09:52,473 INFO    SenderThread:4139 [sender.py:send_request_defer():609] handle sender defer: 14
2023-12-03 23:09:52,474 DEBUG   HandlerThread:4139 [handler.py:handle_request():146] handle_request: poll_exit
2023-12-03 23:09:52,474 DEBUG   HandlerThread:4139 [handler.py:handle_request():146] handle_request: server_info
2023-12-03 23:09:52,474 DEBUG   HandlerThread:4139 [handler.py:handle_request():146] handle_request: get_summary
2023-12-03 23:09:52,474 DEBUG   HandlerThread:4139 [handler.py:handle_request():146] handle_request: sampled_history
2023-12-03 23:09:52,474 DEBUG   SenderThread:4139 [sender.py:send_request():407] send_request: poll_exit
2023-12-03 23:09:52,475 DEBUG   HandlerThread:4139 [handler.py:handle_request():146] handle_request: internal_messages
2023-12-03 23:09:52,475 DEBUG   SenderThread:4139 [sender.py:send_request():407] send_request: server_info
2023-12-03 23:09:52,477 DEBUG   HandlerThread:4139 [handler.py:handle_request():146] handle_request: job_info
2023-12-03 23:09:52,524 DEBUG   SenderThread:4139 [sender.py:send_request():407] send_request: job_info
2023-12-03 23:09:52,525 INFO    MainThread:4139 [wandb_run.py:_footer_history_summary_info():3599] rendering history
2023-12-03 23:09:52,525 INFO    MainThread:4139 [wandb_run.py:_footer_history_summary_info():3631] rendering summary
2023-12-03 23:09:52,525 INFO    MainThread:4139 [wandb_run.py:_footer_sync_info():3558] logging synced files
2023-12-03 23:09:52,526 DEBUG   HandlerThread:4139 [handler.py:handle_request():146] handle_request: shutdown
2023-12-03 23:09:52,526 INFO    HandlerThread:4139 [handler.py:finish():866] shutting down handler
2023-12-03 23:09:53,478 INFO    WriterThread:4139 [datastore.py:close():294] close: /root/dev/Merlion/wandb/run-20231203_230924-Best-AutoEncoder-ts_0-2023-12-03_23-09-22/run-Best-AutoEncoder-ts_0-2023-12-03_23-09-22.wandb
2023-12-03 23:09:53,525 INFO    SenderThread:4139 [sender.py:finish():1534] shutting down sender
2023-12-03 23:09:53,525 INFO    SenderThread:4139 [file_pusher.py:finish():175] shutting down file pusher
2023-12-03 23:09:53,525 INFO    SenderThread:4139 [file_pusher.py:join():181] waiting for file pusher
